clusterloader2/cmd/clusterloader.go

```diff
@@ -18,6 +18,7 @@ package main
 
 import (
 	"fmt"
+	"k8s.io/perf-tests/clusterloader2/pkg/modifier"
```

Comment:
This should go below, but I see it was like that in the origin PR: https://github.com/kubernetes/perf-tests/pull/1247